# Originally from: ncurses 6.0 - patch 20150810 - Thomas E. Dickey
# See: http://invisible-island.net/ncurses/ (-> Development ncurses patches since previous release)
#
# Debian's `mawk` package does not support POSIX RE character classes,
# so patch the `MKlib_gen.sh` script to use TAB+SPACE instead. 
#
Index: ncurses/base/MKlib_gen.sh
Prereq:  1.50 
--- ncurses-6.0-20150808+/ncurses/base/MKlib_gen.sh	2015-08-07 00:48:24.000000000 +0000
+++ ncurses-6.0-20150810/ncurses/base/MKlib_gen.sh	2015-08-10 08:56:39.000000000 +0000
@@ -72,7 +72,7 @@
 # appears in gcc 5.0 and (with modification) in 5.1, making it necessary to
 # determine if we are using gcc, and if so, what version because the proposed
 # solution uses a nonstandard option.
-PRG=`echo "$1" | $AWK '{ sub(/^[[:space:]]*/,""); sub(/[[:space:]].*$/, ""); print; }' || exit 0`
+PRG=`echo "$1" | $AWK '{ sub(/^[ 	]*/,""); sub(/[ 	].*$/, ""); print; }' || exit 0`
 FSF=`"$PRG" --version 2>/dev/null || exit 0 | fgrep "Free Software Foundation" | head -n 1`
 ALL=`"$PRG" -dumpversion 2>/dev/null || exit 0`
 ONE=`echo "$ALL" | sed -e 's/\..*$//'`
